src/lib/FlightTasks/tasks/Orbit/FlightTaskOrbit.cpp

```diff
@@ -173,8 +180,39 @@ bool FlightTaskOrbit::update()
 
 	Vector2f center_to_position = Vector2f(_position) - _center;
 
-	// make vehicle front always point towards the center
-	_yaw_setpoint = atan2f(center_to_position(1), center_to_position(0)) + M_PI_F;
+	switch (_yaw_behavior) {
+	case 0:
+		// make vehicle front always point towards the center
+		_yaw_setpoint = atan2f(center_to_position(1), center_to_position(0)) + M_PI_F;
```

Comment:
This is doing the same as the default implementation.
Can you just put it at the end e.g. 
```
case 0:
default:
	// make vehicle front always point towards the center
	_yaw_setpoint = atan2f(center_to_position(1), center_to_position(0)) + M_PI_F;
```
I would personally leave away the error message since it's clearly defined what does what and there's a clear default.